pkgs/by-name/wa/wayvr-dashboard/package.nix

```diff
@@ -54,15 +56,21 @@ rustPlatform.buildRustPackage {
 
     nodejs
     importNpmLock.npmConfigHook
+
+    wrapGAppsHook4
   ];
 
   buildInputs = [
-    glib
-    gtk3
-    gtk4
     webkitgtk_4_1
-    libsoup_3
+    glib
+    gdk-pixbuf
+    librsvg
+
+    openssl
     alsa-lib
+
+    # for pactl_wrapper.rs
+    pulseaudio
```

Comment:
I don't think this is going to work, as this will only expose `pactl` to the build process and not the actual application.

There are two ways to add runtime programs to Nix packages:

1. Use a wrapper (using `wrapProgram` in postInstall)
2. Patch the binary path in the source code

For now the cleanest option should be the first one.

Maybe if upstream adds a constant somewhere for the binary name/path, we could patch that out in the future